api/AuthenticatorAssertionResponse.json

```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "AuthenticatorAssertionResponse": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AuthenticatorAssertionResponse",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
```

Comment:
Currently in Canary without a flag (67) and has been announced that it will be 65 behind a flag:

https://groups.google.com/a/fidoalliance.org/forum/#!searchin/fido-dev/christiaan%7Csort:date/fido-dev/VNQwv0aH-vM/HIdnFWOWBAAJ

Reply:
added